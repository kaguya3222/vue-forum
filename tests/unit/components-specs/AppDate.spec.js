import { mount } from "@vue/test-utils";
import AppDate from "../../../src/components/AppDate";

describe("AppDate", () => {
  test("Shows date in right format", () => {
    const wrapper = mount(AppDate, {
      propsData: {
        unixDate: 1582302580890
      }
    });
    expect(wrapper).toMatchSnapshot();
  });
});
