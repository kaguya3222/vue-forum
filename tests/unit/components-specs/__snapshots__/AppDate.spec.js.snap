// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppDate Shows date in right format 1`] = `
<span title="March 5th 52111, 8:41:30 am" class="post-date text-faded">
  in 50091 years
</span>
`;
